src/exportJarFileCommand.ts

```diff
@@ -0,0 +1,337 @@
+        const extension: Extension<any> | undefined = extensions.getExtension("redhat.java");
+        const extensionApi: any = await extension?.activate();
+        const outClassPaths: string[] = [];
+        const setUris: Set<string> = new Set<string>();
+        let pickDependencies: IJarQuickPickItem[] = [];
+        const pickedDependencies: IJarQuickPickItem[] = [];
+        for (const rootNode of rootNodes) {
+            const classPaths: ClasspathResult = await extensionApi.getClasspaths(rootNode.uri, { scope: "runtime" });
+            pickDependencies = pickDependencies.concat(generateDependencies(classPaths.classpaths, setUris, projectPath, true));
+            pickDependencies = pickDependencies.concat(generateDependencies(classPaths.modulepaths, setUris, projectPath, true));
+            const classPathsTest: ClasspathResult = await extensionApi.getClasspaths(rootNode.uri, { scope: "test" });
+            pickDependencies = pickDependencies.concat(generateDependencies(classPathsTest.classpaths, setUris, projectPath, false));
+            pickDependencies = pickDependencies.concat(generateDependencies(classPathsTest.modulepaths, setUris, projectPath, false));
+        }
+        if (pickDependencies.length === 0) {
+            return reject("No class path found.");
+        } else if (pickDependencies.length === 1) {
+            outClassPaths.push(pickDependencies[0].uri);
+            return resolve(outClassPaths);
+        }
+        pickDependencies.sort((node1, node2) => {
+            if (node1.description !== node2.description) {
+                return node1.description.localeCompare(node2.description);
+            }
+            if (node1.type !== node2.type) {
+                return node2.type.localeCompare(node1.type);
+            }
+            return node1.label.localeCompare(node2.label);
+        });
+        for (const pickDependency of pickDependencies) {
+            if (pickDependency.picked) {
+                pickedDependencies.push(pickDependency);
+            }
+        }
+        const pickBox = createPickBox("Export Jar - Determine elements", "Select the elements...", pickDependencies, true, pickSteps.length);
+        pickBox.selectedItems = pickedDependencies;
+        pickBox.onDidTriggerButton((item) => {
+            if (item === QuickInputButtons.Back) {
+                reject(InputFlowAction.back);
+                pickBox.dispose();
+            }
+        });
+        pickBox.onDidAccept(() => {
+            for (const item of pickBox.selectedItems) {
+                outClassPaths.push(item.uri);
+            }
+            resolve(outClassPaths);
+            pickBox.dispose();
+        });
+        pickBox.onDidHide(() => {
+            reject();
+            pickBox.dispose();
+        });
+        pickBox.show();
+    });
+}
+
+function createPickBox(title: string, placeholder: string, items: IJarQuickPickItem[],
```

Comment:
1. pickStepLength: number -> backBtnEnabled: boolean
2. canSelectMany should have default value